### ISSUE TITLE
Unable to visualize the output layer of MobileNet V1 SSD model

### ISSUE THREAD

**Issue by AkkiSony** [tensorflow/models#10224]
Hi,

I always use https://netron.app/ (web based IDE) to visualize the model. Just like that, I wanted to know the output layer of my model which was trained with Object Detection API. The other models such as darknet was easy for me to find out the output layer, but I find it difficult with this model.

Can you please help me understand the output layyer of the model? Please find the link attached for the model.  

https://drive.google.com/drive/folders/1pq8DTzc_b4CWIf65Y26CIzxQxg_WoL2c?usp=sharing

**I need the name of the output layer, which further helps me convert the model into the format, I need it to be. Hence, please let me know if you could find the output layer of this model.**

For example, I need to mention like this. But I do not know my output layer. :(

$convert_tf \
    --tf-pb ./model/mobilenet_v1.pb \
    --inputs input \
    --input-size-list '224,224,3' \
    --outputs MobilenetV1/Logits/SpatialSqueeze \

Thanks in advance. :)

**Comment by AkkiSony** [comment]
Thanks for your reply! :)
